driver/compmisc.ml

```diff
@@ -57,8 +57,16 @@ let initial_env () =
     else Env.initial_safe_string
   in
   let env =
-    if !Clflags.nopervasives then initial else
-    open_implicit_module "Pervasives" initial
+    if !Clflags.nopervasives then initial else begin
+      let include_dirs = !Clflags.include_dirs in
```

Comment:
Shouldn't you rather use Config.load_path (assuming it is already initialized from include_dirs at this point)?  include_dirs does not necessarily include the directory of the compiled unit, which is automatically added to load_path (in Compmisc.init_path).

Reply:
Indeed, it should be `load_path`.